website/content/en/status/report-2023-04-2023-06/freebsd-foundation.adoc

```diff
@@ -0,0 +1,169 @@
+In this update, we will outline our contributions to FreeBSD across multiple domains.
+We will touch upon project development initiatives, some of which have detailed reports of their own.
+Additionally, we will showcase our advocacy for FreeBSD, our efforts to foster community engagement, and our expansion of partnership endeavors.
+Lastly, we will delve into our ongoing work to secure increased funding, enabling us to allocate additional resources to address any gaps within the Project.
+
+==== Fundraising
+
+During the previous quarter, we made significant progress in engaging with commercial FreeBSD users.
+To enhance our partnerships with existing and potential commercial users, we hired Greg Wallace as the Director of Partnerships and Research.
+His primary objective is to expand our collaborations with commercial users.
+Since assuming this position, Greg has hit the ground running, meeting with numerous companies in just one quarter.
+These interactions have provided valuable insights into how FreeBSD is being utilized, the challenges faced by users, and areas where the Project can improve.
+By understanding these aspects, we can make informed decisions on where to allocate our funding and recognize FreeBSD's unique strengths.
+Additionally, the role involves conducting research to identify target markets, explore new opportunities for FreeBSD, and ensure our voice is heard in relevant discussions.
+For more details on Greg's objectives and accomplishments in the previous quarter, you can refer to his status update below.
+
+The Foundation extends its heartfelt gratitude to everyone who made financial contributions to support our work during the previous quarter.
+Besides many individual contributions, we were pleased to receive larger donations from NetApp and Blackberry.
+In addition,  we received FreeBSD Developer Summit sponsorships from Tarsnap, iXsystems, and LPI.
+These sponsorships greatly assist in offsetting our expenses and enable us to offer affordable registration fees to attendees.
+
+This year our budget is around $2,230,000, which includes increased spending toward FreeBSD advocacy and software development.
+More than half of our budget is allocated toward work directly related to improving FreeBSD and keeping it secure.
+
+By having a dedicated individual focused on partnerships, we can effectively emphasize the significance of investing in our efforts and underscore the long-term viability of the Project to companies.
+
+Your support plays a crucial role in our mission, and we deeply appreciate your commitment to the FreeBSD community.
+Please consider making a donation toward our 2023 fundraising campaign!
+link:https://www.freebsdfoundation.org/donate/[]
+
+We also have a Partnership Program for more prominent commercial donors.
+You can read about it at link:https://freebsdfoundation.org/our-donors/freebsd-foundation-partnership-program/[].
+
+==== Partnership Program
+
+Hello FreeBSD community.
+I'm Greg Wallace.
```

Comment:
No contractions. (Also fixable on commit.)

Reply:
As the first/second line of a personal introduction, I think the contraction is OK. 

Expanded, here, might sound slightly robotic.

Reply:
I think we should not overthink it, I am fine both with and without the contraction in this case.

I am also fine with "My name is", in case this is perceived as not too formal while it also avoids the contraction.